config/intelfocl_sample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "TARGET" : "intelfocl",
+  "HW_VER" : "0.0.1",
```

Comment:
We might need to update this version number, since the HW ISA has changed?

Reply:
Agreed. Let bump it up to 0.0.2

Reply:
Please apply the 0.0.2 bump thanks